new-site/src/components/layout.scss

```diff
@@ -177,7 +177,17 @@ body {
 
 .headerAnchor {
   color: var(--color-black-90);
-  margin-left: 3px;
+  padding-left: 0 !important;
+  margin-left: 7px;
```

Comment:
>   margin-left: 7px;

Very minor nagging but maybe use `--spacing-2-xs` (8px) here for consistency?

Reply:
- [ ] Use `--spacing-2-xs` for consistency